spec/factories/users.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
+    username { Faker::Internet.user_name }
+    password_digest { Faker::Internet.password }
+  end
+
+  # factory :user do
```

Comment:
No comments. Comments are just future lies. If you need to save something, I recommend putting it in a branch.